speculation-rules.bs

```diff
@@ -43,6 +45,12 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
       text: rules for choosing a navigable; url: the-rules-for-choosing-a-navigable
     urlPrefix: semantics.html
       text: get an element's target; url: get-an-element's-target
+    urlPrefix: links.html
+      text: following a hyperlink; url: following-hyperlinks-2
+  type: attr-value
+    urlPrefix: links.html
+      for: a/rel
+        text: noreferrer; url: link-type-noreferrer
```

Comment:
Hmm, this shouldn't be necessary; HTML has it marked up correctly. But `bikeshed refs --text noreferrer --type attr-value` doesn't give any results. So, not sure what's going on there. No action for you.

Reply:
Ah, this is https://github.com/tabatkins/bikeshed/issues/1752.